lib/web/users_test.go

```diff
@@ -118,6 +119,186 @@ func TestCRUDs(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestSetTrait(t *testing.T) {
```

Comment:
Maybe:

```suggestion
func TestUpdateUser_setTraits(t *testing.T) {
```

So it's clear which endpoint we are operating on.